x-pack/platform/plugins/private/upgrade_assistant/server/plugin.ts

```diff
@@ -119,6 +119,8 @@ export class UpgradeAssistantServerPlugin implements Plugin {
     });
 
     const router = http.createRouter();
+    // Initialize version service with current kibana version
+    versionService.setup(this.kibanaVersion);
```

Comment:
Using the version service apis in the route dependencies meant reshuffling the code a bit.